### DISCUSSION TITLE
Following the installation guide for MacOS, does not work

### DISCUSSION THREAD

**Discussion by meglio** [basecamp/kamal#495] [Q&A]
<img width="1157" alt="image" src="https://github.com/basecamp/kamal/assets/282177/429e03c8-1192-41ac-9ee5-d3b5c6aab80a" scale=10>

When I run `kamal init` in MacOS, it says:

<img width="505" alt="image" src="https://github.com/basecamp/kamal/assets/282177/a1e9563e-4992-4465-b4b3-d8150a9ef3f4">

**Comment by meglio** [comment]
The alias that worked for me:

`alias kamal="docker run -it --rm -v \"\${PWD}:/workdir\" -v '/run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock' -e SSH_AUTH_SOCK='/run/host-services/ssh-auth.sock' -v '/var/run/docker.sock:/var/run/docker.sock' ghcr.io/basecamp/kamal:latest"`

... in case you wanted to update the documentation.

**Reply by LondonAppDev** [reply]
Thanks for sharing this. I had exactly the same problem. Basically with the way it's documented, the config file and hooks are created in the home directory.
